Dockerfile

```diff
@@ -1,23 +1,22 @@
-FROM python:3.5
+FROM python:3.6.2-slim
```

Comment:
Just out of curiosity: any reason to pick up `slim` in favor of `alpine`? Testing this PR locally and gonna merge if it works properly. Just asking about the distro out of curiosity ; )

Reply:
Slim is based on debian

Reply:
And why being based on Bebian is better/desired/required, @caduvieira? Just so we don't have to find equivalent packages in `apk`?